src/mono/wasm/runtime/library_mono.js

```diff
@@ -2243,7 +2243,7 @@ var MonoSupportLib = {
 			manifest.filter(m => {
 				var file = m[0];
 				var last = file.lastIndexOf ("/");
-				var directory = file.slice (0, last);
+				var directory = file.slice (0, last+1);
```

Comment:
What is this for?

Reply:
For timezone data files not already nested in the directory. The last character gets cut off and it's fine for the other directories to have a `/` at the end.

Reply:
But what changed in the PR to require this?

Reply:
Additional files are included from runtime-assets.

Reply:
This confuses me too, can you give an example of a manifest line that was failing?

Reply:
It was failing when creating the `/usr/share/zoneinfo/GMT` data file. There is another data file named `GMT0` and without the +1 a directory named `/usr/share/zoneinfo/GMT` is already created beforehand. It wasn't failing before because we didn't have this naming collision since we only had `zone.tab` in the root directory.

Reply:
So did the paths in the manifest change from "/GMT" to "GMT" when the code moved to runtime-assets?

Reply:
The file GMT was added in runtime-assets. We used to filter it out in the build task.